resources/eventing/values.yaml

```diff
@@ -5,11 +5,11 @@ global:
   images:
     eventing_controller:
       name: eventing-controller
-      version: PR-14580
+      version: PR-14599
```

Comment:
Is this a test ? :-D
Doesn't need to be updated but why not :-D

Reply:
uuups